.devcontainer/devcontainer.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Jimmi",
+   "build": {
+    "dockerfile": "Dockerfile",
+    "args": {
+      "VARIANT": "bullseye"
+    }
+  },
+  "settings": {
+    "deno.enable": true,
+    "deno.lint": true,
+    "editor.defaultFormatter": "denoland.vscode-deno"
+  },
+  "extensions": [
+    "denoland.vscode-deno",
+    "svelte.svelte-vscode"
+  ],
+  "postStartCommand": "vr start",
```

```suggestion
  "postStartCommand": "trex run start",
```
We might consider using trex instead, in case the run script changes in the future